server_api.py

```diff
@@ -244,9 +244,9 @@ def set_insert_customer_text_msg():
         send_msg["server_dir"] = ""
         send_msg["file_type"] = display_data_type(type_name='customized_text')[0]
         send_msg["start_date"] = time.strftime("%Y-%m-%d", time.localtime(now_time))
-        send_msg["end_date"] = time.strftime("%Y-%m-%d", time.localtime(now_time+3))
+        send_msg["end_date"] = time.strftime("%Y-%m-%d", time.localtime(now_time+9))
         send_msg["start_time"] = time.strftime("%H:%M:%S", time.localtime(now_time))
-        send_msg["end_time"] = time.strftime("%H:%M:%S", time.localtime(now_time+3))
+        send_msg["end_time"] = time.strftime("%H:%M:%S", time.localtime(now_time+9))
```

Comment:
3 seconds life time is not enough.
When arrange_schedule is going to show the personal page, it's expired rate is 40%, most of activity display 5 seconds, normally.